google_analytics/ga_homepage_report.py

```diff
@@ -302,7 +305,8 @@ def clicksDataframe(clicks_data):
     clicks_dataframe = DataFrame(clicks_data, columns=['date', 'cardName', 'position', 'totalClicks', 'uniqueClicks'])
     clicks_dataframe = clicks_dataframe.apply(to_numeric, errors='ignore')
     clicks_dataframe.drop('date', axis=1, inplace=True)
-    clicks_dataframe = clicks_dataframe.groupby(['cardName','position']).sum().sort_values(by='uniqueClicks',ascending=0)
+    clicks_dataframe = clicks_dataframe.groupby(['cardName', 'position']).sum().sort_values(by='uniqueClicks',
```

Comment:
As an alternative, we could shorten this line by moving the groupby arg into a named variable on the line above.
```
group_cols = ['cardName', 'position']
clicks_dataframe = clicks_dataframe.groupby(group_cols).sum().sort_values(by='uniqueClicks',ascending=0)
```